TarSCM/scm/git.py

```diff
@@ -201,11 +201,37 @@ def _detect_version_parent_tag(self, parent_tag, versionformat):  # noqa pylint:
                                versionformat)
         return versionformat
 
+    def check_ancestry(self, parent_tag):
+        rcode, tgcommit = self.helpers.run_cmd(
+            ["git", "rev-parse", parent_tag + "^{}"],
+            self.clone_dir
+        )
+        if rcode:
+            sys.exit("\033[31m`git rev-parse " + parent_tag + "` failed: " +
+                     output + "\033[0m")
```

Comment:
Travis [correctly spotted](https://travis-ci.org/openSUSE/obs-service-tar_scm/jobs/395050912) that `output` is not defined here.